notebooks/4_plotting.ipynb

```diff
@@ -0,0 +1,512 @@
+{
```

Comment:
<p>For this diagram, I'm not sure what to make of the blue and the green. On the left side, there are two green columns and one blue column. On the right, I see two blue plots (one of which has green lines) and one green plot. Am I supposed to make a connection between the left and the right? If not, maybe tweak the colors to be different.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>The idea is actually that the single blue column is used to create a histogram and a boxplot: summary plots of a single column. The scatter plot provides  the representation of 2 columns. Would it suffice to make the small green lines of the boxplots grey or would this still be confusing? Maybe just no coloring of columns would be the best?  Other suggestions to improve this certainly welcome.</p>